android/uhabits-android/src/main/java/org/isoron/uhabits/receivers/WidgetReceiver.java

```diff
@@ -67,7 +67,6 @@ public void onReceive(final Context context, Intent intent)
 
         IntentParser parser = app.getComponent().getIntentParser();
         WidgetBehavior controller = component.getWidgetController();
-        Preferences prefs = app.getComponent().getPreferences();
```

Comment:
Please revert this, unless it is required. Refactoring is welcome, but please submit it as a separate PR.